docs/setup/independent/high-availability.md

```diff
@@ -490,8 +490,8 @@ Next provision and set up the worker nodes. To do this, you will need to provisi
 1. Reconfigure kube-proxy to access kube-apiserver via the load balancer:
 
     ```shell
-    kubectl get configmap -n kube-system kube-proxy -o yaml > kube-proxy.yaml
-    sudo sed -i 's#server:.*#server: https://<masterLoadBalancerFQDN>:6443#g' kube-proxy.cm
+    kubectl get configmap -n kube-system kube-proxy -o yaml > kube-proxy.cm
```

Comment:
I'd rather stick to `kube-proxy-cm.yaml`, as we don't use kind in filename extensions.